setup.py

```diff
@@ -111,6 +111,7 @@ def get_git_sha():
         "sqlalchemy>=1.3.16, <1.4, !=1.3.21",
         "sqlalchemy-utils>=0.36.6,<0.37",
         "sqlparse==0.3.0",  # PINNED! see https://github.com/andialbrecht/sqlparse/issues/562
+        "tabulate==0.8.9",
```

Comment:
You will have to [update](https://github.com/apache/superset/blob/master/CONTRIBUTING.md#dependencies) the requirements. Additionally strict pinning of packages in `setup.py` is undesirable as it may overly constrain the frozen solution space. Please use limits if strictly necessary.

Reply:
@john-bodley I did run `pip-compile-multi --no-upgrade`, you can see that `requirements/base.txt` was updated to include the dependency (it has no additional dependencies of its own).

Also, my understanding is that it's better to pin the version for dependencies with version `0.x`, since there's no guarantee that a minor upgrade will maintain backwards compatibility (https://semver.org/#spec-item-4).